src/storage/src/source/postgres.rs

```diff
@@ -437,7 +444,7 @@ async fn postgres_replication_loop_inner(
 struct RowSender {
```

Comment:
god we really need to make this not a tokio task and to just happen in the `next` function; its even more important now because we are doing the cast on the tokio thread and should be doing it on the timely thread